format/Flight.proto

```diff
@@ -301,6 +301,44 @@ message Location {
  */
 message Ticket {
   bytes ticket = 1;
+  
+  // Data representing some part of the data retrievable by the ticket.
+  //
+  // This is an optional optimization for applications that want to reduce latency,
+  // for previewing results or retrieving small data sets by providing data
+  // without requiring a DoGet call.  Applications built on top of Flight 
+  // are responsible for any negotiation necessary on whether
+  // inlining data is appropriate.
+  //
+  // The size of inlined_data is expected to be small (typically less then 1MB) 
+  // and inlining too much data across tickets can run into underlying transport 
+  // limitations.  Furthermore, since the data is expected to be small, implementations 
+  // are less likely to optimize for zero-copy in these cases. 
+  message InlinedData { 
+    // Arrow data for consumption.
+    //
+    // The data is not expected to contain schema metadata. The schema
```

Comment:
I guess nit, but maybe 'not expected to contain the schema message'?

Reply:
done.